.gitignore

```diff
@@ -2,3 +2,6 @@ out
 node_modules
 format.patch
 Makefile
+
+# production
```

Comment:
as this is for `read` can you revise this to `# production for react-app` or meaningful?

Reply:
thank you for feedback : )